### DISCUSSION TITLE
chia1.2Can't connect to wallet

### DISCUSSION THREAD

**Discussion by bxunnd** [guydavis/machinaris#166] [Q&A] [answered]
chia1.2 Can't connect to wallet
1.1.7no problem
With Windows 7

**Comment by guydavis** [comment, marked as answer]
Hi, please restart the Machinaris container.  Also, recommend upgrading to Windows 10, as 7 is EOL for over a year now... unsupported.

**Reply by guydavis** [reply]
Good day, it's been a couple of days.  Did my suggestion help?  If so, please mark the Discussion as Answered.  If not, please post further details, including [all logs](https://github.com/guydavis/machinaris/wiki/Troubleshooting#nothing-works--where-are-the-logs) along with your exact Docker run/compose command.

**Reply by guydavis** [reply]
No response.  Marking as Answered.
